src/lib/ui/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -116,6 +117,13 @@ function AccessCodeRow(props: {
   onClick: () => void
 }) {
   const { onClick, accessCode } = props
+
+  const errorCount = accessCode.errors?.length ?? 0
```

Comment:
Interesting, I didn't think `errors` could be null, I thought it was always at least an empty array.

Reply:
![Code 2023-05-30 at 15 08 52@2x](https://github.com/seamapi/react/assets/87919558/c28b6e7e-d09e-4eb4-acb7-b50695a9cc7c)

Reply:
Ah ok, I was thinking about `Device` where it is non-null! looks like for AccessCode it can be undefined 👍🏻